server/lib/get-sent-messages.js

```diff
@@ -0,0 +1,6 @@
+const { getMessages } = require('./db')
+const { messageStates: { sent } } = require('../constants')
+
+module.exports = async () => {
+  return getMessages(`SELECT * FROM c WHERE c.state = "${sent}" ORDER BY c.lastUpdatedAt DESC`)
```

Comment:
I considered adding this to `./db` but didn't as it is a bit more specialised than the functions that module should be providing.